src/test/java/bio/terra/app/controller/SnapshotValidationTest.java

```diff
@@ -31,7 +31,7 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest
 @AutoConfigureMockMvc
-@ActiveProfiles("google")
+@ActiveProfiles({"google", "connectedtest"})
 @Category(Unit.class)
```

Comment:
Is there a particular connectedtest application property that this Unit test needs?

Reply:
Weird. No idea. I'll revert it.